netci.groovy

```diff
@@ -799,6 +806,12 @@ def static isValidPrTriggeredInnerLoopJob(os, architecture, configuration, isBui
     return true
 }
 
+// This means the job builds and runs the 'Pri0' test set. This does not mean the job is 
+// scheduled with a default PR trigger despite the correlation being true at the moment.
+def static isPri0TestScenario(def scenario) {
```

Comment:
I'm ok having this function, despite my comment above about the correspondence between "innerloop" in the name, and pri-0 tests getting run.